pkg/webhook/indexeres/indexer.go

```diff
@@ -73,3 +78,12 @@ func VolumeByReplicaCount(obj *lhv1beta2.Volume) ([]string, error) {
 	replicaCount := strconv.Itoa(obj.Spec.NumberOfReplicas)
 	return []string{replicaCount}, nil
 }
+
+func imageByExportSourcePVC(obj *harvesterv1.VirtualMachineImage) ([]string, error) {
+	if obj.Spec.SourceType != longhorntypes.LonghornLabelExportFromVolume ||
+		obj.Spec.PVCNamespace == "" || obj.Spec.PVCName == "" {
+		return []string{}, nil
```

Comment:
```suggestion
		return nil, nil
```

I usually prefer a nil slice here since I can never remember if the compiler ends up heap-allocating anything for `[]string{}`  and with `nil` I don't have to finally learn to remember :laughing:

Reply:
Updated per suggestion, thanks.